runtime/assembly/__tests__/mock-builder.ts

```diff
@@ -64,6 +64,17 @@ export namespace MockBuilder {
         ];
         return new MockResult(MockHelper._getExtrinsicInstance(), DEFAULT_EXTRINSIC);
     }
+
+    export function getInvalidExtrinsic(): u8[] {
+        const INVALID_EXTRINSIC: u8[] = [
```

Comment:
can you make the invalid extrinsic a little bit more prominent?
At first this extrinsic looked alright to me. I had to check the length of the array to see that it is 143 numbers.

You can create the array to be for example 10 or 15 elements and add comment that the minimum required length is 144